sdk/identity/Azure.Identity.Broker/src/Azure.Identity.Broker.csproj

```
@@ -6,14 +6,15 @@
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
     <ApiCompatVersion>1.2.0</ApiCompatVersion>
     <PackageTags>Microsoft Azure Identity Broker;$(PackageCommonTags)</PackageTags>
-    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Windows'))">$(RequiredTargetFrameworks);net462;net6.0</TargetFrameworks>
-    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('OSX'))">$(RequiredTargetFrameworks);net6.0</TargetFrameworks>
-    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Linux')) ">$(RequiredTargetFrameworks);net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Windows'))">$(RequiredTargetFrameworks);net462;net8.0</TargetFrameworks>
+    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('OSX'))">$(RequiredTargetFrameworks);net8.0</TargetFrameworks>
+    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Linux')) ">$(RequiredTargetFrameworks);net8.0</TargetFrameworks>
```

Any reason not to use these simpler OS detection methods?

```suggestion
    <TargetFrameworks Condition="$([MSBuild]::IsWindows())">$(RequiredTargetFrameworks);net462;net8.0</TargetFrameworks>
    <TargetFrameworks Condition="$([MSBuild]::IsMacOS())">$(RequiredTargetFrameworks);net8.0</TargetFrameworks>
    <TargetFrameworks Condition="$([MSBuild]::IsLinux()) ">$(RequiredTargetFrameworks);net8.0</TargetFrameworks>
```

---

no reason that I'm aware of.